application.yml

```diff
@@ -88,4 +88,8 @@ spring:
     init:
       schema-locations: classpath*:db/mysql/schema.sql
       data-locations: classpath*:db/mysql/data.sql
-      init: ALWAYS
\ No newline at end of file
+      mode: ALWAYS
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.MySQLDialect
```

Comment:
Could you please check if the dialect is required?
Hibernate 6 provides automatic Hibernate Dialect resolution: https://vladmihalcea.com/hibernate-dialect/

Reply:
Thanks @arey, You are right, I checked again and it works without explicit configuration of dialect. I wonder why that didn't work for me the first time... welp, false alarm then :)

Sorry I took long to reply, I just realized today I had my old email set as a default.

Reply:
Ok thank you for your feedback @lemonise 
I'll close this pull request.